### DISCUSSION TITLE
How do I get a track by ISRC code

### DISCUSSION THREAD

**Discussion by Maxzeno** [tidal-music/discussions#26] [Q&A] [answered]
How do I get a track by ISRC code

**Comment by stefanmoro** [comment]
Hey @Maxzeno.

It is currently not possible to get tracks for a certain ISRC. Such an endpoint would return a list of TIDAL tracks, similar to what the endpoint returning albums by UPC does, see #21 for more info.

Could you help us better understand the use case, something that would be valuable when looking in to if such functionality would be possible/feasible to build.

Thanks

**Comment by aliaksei-taliuk** [comment, marked as answer]
@Maxzeno Hi!

Happy to announce that we've added a new endpoint that should solve your problem. You can find more details on this API Reference page: https://developer.tidal.com/apiref?spec=catalogue&ref=get-tracks-by-isrc 

Thanks

**Reply by SubSploink** [reply]
As the link is dead, the current API call to be able ot search for a track by ISRC seems to be here:
https://developer.tidal.com/apiref?spec=catalogue-v2&ref=get-all-tracks&at=THIRD_PARTY

**Reply by allie-hawthorne** [reply]
New new link is here: https://tidal-music.github.io/tidal-api-reference/#/tracks/get_tracks
See Britnell's answer below for more details in case this link is dead too lol

**Comment by JenBytes** [comment]
Hey, are there any updates regarding this? 
Is there still an endpoint or another way to get a track by ISRC?

**Comment by Britnell** [comment]
not sure why the questions adn the dead links
https://tidal-music.github.io/tidal-api-reference/#/tracks/get_tracks

the endpoint is /tracks incase this link breaks too, 
 use `filter[isrc]` query param
sdk : 
```
apiClient
    .GET('/tracks', {
      params: {
        query: {
          countryCode,
          include: ['albums', 'artists'],
          'filter[isrc]': ids,
        },
      },
    })
```
